packages/client/src/generation/generateClient.ts

```diff
@@ -405,3 +405,16 @@ function validateDmmfAgainstDenylists(prismaClientDmmf: PrismaClientDMMF.Documen
 
   return errorArray.length > 0 ? errorArray : null
 }
+
+export function getPackageJsonName(outputPath: string): string {
```

Comment:
For me, `PackageJsonName` implies one of: 

```
"name": "@prisma/sdk",
```

```
"name": "@prisma/migrate",
```

```
"name": "@prisma/client",
```

```
"name": "xyz",
```

Is this function getting that `.name` value inside the package json?